resources/splunk-dashboards/README.md

```diff
@@ -0,0 +1,28 @@
+
+## @dotcom-reliability-kit/splunk-dashboard
+
+Splunk dashboard templates for apps which use [FT.com Reliability Kit](https://github.com/Financial-Times/dotcom-reliability-kit#readme).
+
+  * [Usage](#usage)
+  * [Contributing](#contributing)
+  * [License](#license)
+
+
+## Usage
+
+This package includes JSON files which can be used to create Splunk dashboards. The following dashboard templates are available:
```

Comment:
Done 👍 is this clear enough? I really don't want to end up maintaining full instructions with screenshots and everything because they go out of date so quickly 😄

Reply:
Perfect - thank you 🙇